resources/assets/js/markdown-editor/css/style.css

```diff
@@ -244,4 +244,23 @@
     color: inherit;
 }
 
+.UserTagger .highlighted .UserTagger--username {
+    @apply text-theme-primary-600;
+}
+.UserTagger > ul > li {
+    @apply mx-6;
+}
+.UserTagger > ul > li:not(:last-child) {
+    @apply border-b border-dotted border-theme-secondary-300;
+}
+
+.UserTagger > ul > li.highlighted {
+    @apply px-6 mx-0 border-transparent bg-theme-primary-100;
+}
+
+[contenteditable="true"]:empty:before {
```

Comment:
should this be made more specific so it won't interfere with other elements with `contenteditable`? 🤔